domain/src/test/java/camp/nextstep/edu/domain/calculator/ExpressionParserTest.kt

```diff
@@ -0,0 +1,99 @@
+package camp.nextstep.edu.domain.calculator
+
+import org.junit.Assert.*
+import org.junit.Test
+
+
+class ExpressionParserTest {
+
+    private val expressionParser = ExpressionParser()
+
+
+    @Test
+    fun `1 + 1`() {
+        val expression = "1 + 1"
+        val parsed = expressionParser.parse(expression)
+        assertEquals(
+            listOf(Num(1), Operators.Plus, Num(1)),
+            parsed
+        )
+    }
+
+    @Test
+    fun `1 - 1`() {
+        val expression = "1 - 1"
+        val parsed = expressionParser.parse(expression)
+        assertEquals(
+            listOf(Num(1), Operators.Minus, Num(1)),
+            parsed
+        )
+    }
+
+    @Test
+    fun `1 * 1`() {
+        val expression = "1 * 1"
+        val parsed = expressionParser.parse(expression)
+        assertEquals(
+            listOf(Num(1), Operators.Multiply, Num(1)),
+            parsed
+        )
+    }
+
+    @Test
+    fun `1 나누기 1`() {
+        val expression = "1 / 1"
+        val parsed = expressionParser.parse(expression)
+        assertEquals(
+            listOf(Num(1), Operators.Divide, Num(1)),
+            parsed
+        )
+    }
+
+    @Test
+    fun `공백은 입력할 수 없다_0`() {
```

Comment:
```suggestion
    fun `빈 문자열을 입력하는 경우 IllegalArgumentException이 발생한다`() {
```
테스트 함수 명을 조금 더 직관적으로 수정하는 것도 좋을 것 같아요!
현재와 같은 테스트명을 가지게 되면 함수명만으로는 어떤 코드가 문제가 되어서 어떤 에러가 발생되려했는지를 알 수가 없습니다.

처음 코드를 설계할 때 외에 코드가 수정이 됐을 경우에도 작성해주신 테스트코드들이 모두 정상적으로 동작한다고 보장할 수가 없게 되는데요.
그 경우 수정을하려면 테스트코드를 모두 다시 읽어서 어떤 내용인지를 파악해야한다는 단점이 생길 것 같아요!